src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class Ball {
+    String color;
+    int number;
+
+    @Override
+    public String toString() {
+        return "Color: " + color+ "\n" + "Number: " + number;

```

Comment:
It's better to avoid using newline characters such as '\n' in the 'toString' method. The 'toString' method is generally used to provide a string representation of the object, and the formatting (including line breaks) should be handled by the code that outputs the object, not the object itself.